packages/contentful/src/crn/queries/manuscript.queries.ts

```diff
@@ -484,7 +484,7 @@ export const FETCH_MANUSCRIPTS_BY_TEAM_ID = gql`
   query FetchManuscriptsByTeamId($id: String!) {
     teams(id: $id) {
       linkedFrom {
-        manuscriptsCollection(limit: 500) {
+        manuscriptsCollection(limit: 500, order: count_DESC) {
```

Comment:
I was taking a look at this query, we grab 500 manuscripts, then check if the team is inside teamsCollection of the manuscript to grab the count.

I think we could have something like this 

```
  manuscriptsCollection(
    where: { teams: { sys: { id: $id } } }
    limit: 1,
    order: count_DESC
  ) {
    items {
      count
    }
  }
```

instead, right?

Reply:
I see, it makes sense. I think we can at least add a `limit: 1` here :)

```    
teams(id: $id) {
  linkedFrom {
    manuscriptsCollection(limit: 500, order: count_DESC) {
      items {
        teamsCollection(limit: 1) {    <---
          items {
            sys {
              id
            }
          }
        }
        count
      }
    }
  }
}
```